### PR TITLE
chore: font updates on buy, wallet disabled view fix

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: brd-ios/breadwallet/src/ViewControllers/LoginViewController.swift [hunk 1]
```diff
@@ -445,6 +445,7 @@ class LoginViewController: UIViewController, Subscriber, Trackable {
             return
         }
         saveEvent("login.locked")
+        navigationController?.isNavigationBarHidden = true
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ViewControllers/LoginViewController.swift:448]: why is this in the lockIfNeeded func

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift [hunk 2]
```diff
@@ -52,7 +52,8 @@ class CardSelectionViewController: ItemSelectionViewController {
         }
         
         cell.setup { view in
-            view.configure(with: .init())
+            let config = CardSelectionConfiguration(title: .init(font: Fonts.Title.six, textColor: LightColors.Icons.one))
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift:55]: do we have a Presenter for this VC?

File: brd-ios/breadwallet/src/ViewControllers/LoginViewController.swift (hunk 1 above)
Comment (dangelovska) [reply brd-ios/breadwallet/src/ViewControllers/LoginViewController.swift:448]: it's only needed when walletDisabled is shown

File: brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift (hunk 2 above)
Comment (dangelovska) [reply brd-ios/breadwallet/src/Redesign/Scenes/Buy/CardSelection/CardSelectionViewController.swift:55]: here we are reusing the ItemSelectionViewController, that's why the config is setup here here

**Review by kenan-mamedoff — APPROVED** [review]

**Pull request merged**